packages/widgets/src/additional-svelte-typings.d.ts.ts

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line @typescript-eslint/no-namespace, @typescript-eslint/no-unused-vars
```

Comment:
too many `.ts` in the filename

Reply:
yep https://github.com/huggingface/huggingface.js/pull/577/commits/dcac7241238a5e510894d51fa0eeb937f9ac6afd